File: Platron/Tests/AddingAnItemToTheBinTest.cs
using System;
using NUnit.Framework;
using NUnit.Allure.Core;
using NUnit.Allure.Attributes;
using Platron.Pages.Helpers;

namespace Platron.Tests
{
    [TestFixture]
    [AllureNUnit]
    [AllureSuite("Platron")]
    public class AddingAnItemToTheBinTest : BaseTest
    {
        [Test]
        public void AddingAnItemToTheBin()
        {
            Print("AddingAnItemToTheBin");
            new DemoshopHelper()
                .ClickInBinOnPage()
                .CheckExistRowsInBin(1);
        }
    }
}

File: Platron/src/test/java/AddingAnItemToTheBinTest.java
import org.junit.Test;

public class AddingAnItemToTheBinTest extends BaseTest {
    @Test
    public void AddingAnItemToTheBin() throws Exception {
//        System.out.println(System.getProperty("user.dir"));
//        System.out.println(System.getProperty(outputPath));
        IHelper er = new DemoshopHelper()
                .ClickInBinOnPage(1)
                .CheckExistRowsInBin(1);
    }
}